modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3421,6 +3421,91 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "LPS-173537 - Verify adding a Commerce Product entry triggers an action to update the Commerce Product Group entry."
+	@priority = 5
+	test CanUpdateCommerceProductGroupEntryAfterCreatingCommerceProductEntry {
+		property portal.acceptance = "true";
+
+		task ("Given: A Commerce Product Group") {
```

Comment:
I forgot the add the "object" to the task. So the goal for this task is to go to Commerce Product Group on Objects.

`task ("Given a Commerce Product Group object")`

**This way:**  

`ObjectAdmin.openObjectAdmin();`

`ObjectPortlet.selectSystemObject(label = "Commerce Product Group");`